exercises/run-length-encoding/src/RunLength.hs

```diff
@@ -0,0 +1,9 @@
+module RunLength (decode, encode) where
+
+import Data.Char (isDigit)
+
+decode :: String -> String
+decode = undefined
+
+encode :: String -> String
+encode = undefined
```

Comment:
This is perfect the way it is and also matches exactly what we did in all the other exercises.

Recently, we discussed in #459 that it would be nice to use `error` to give better messages than `undefined`, so an alternative would be:

```haskell
encode xs  = error "You need to implement this function."
```

No change is needed here, and we are not yet in the processes of rewriting the stub solutions. Your choice!